pandas/tests/io/parser/common.py

```diff
@@ -238,6 +238,21 @@ def test_csv_mixed_type(self):
         out = self.read_csv(StringIO(data))
         tm.assert_frame_equal(out, expected)
 
+    def test_read_csv_low_memory_no_rows_with_index(self):
```

Comment:
I think there is a class that you can put the will specifically only have low_memory set, just search for self.low_memory

Reply:
Indeed, there is, but I deliberately allowed this test to run for both the C and Python engines.  As the Python engine doesn't support `low_memory`, I need to do this.